docker-compose.dev.yaml

```diff
@@ -43,6 +43,6 @@ services:
       - /var/lib/postgresql/data
     healthcheck:
       test: ["CMD-SHELL", "pg_isready"]
-      interval: 10s
+      interval: 15s
```

Comment:
This seems like a good change, but was there a specific reason you found that required this change?